### ISSUE TITLE
Better way to select specific historic entries to edit 

### ISSUE THREAD

**Issue by BenKettlewell** [jrnl-org/jrnl#677]
Jrnl really needs a better way to select which jrnl entries to view/open/edit.

The only way to filter the entries you want is by consecutive timestamps and tags. Tags are amazing but very often come up short. Time entries are useful only if you're using jrnl like a journal (heh).

In the following example I use `@notes` and `@keyword` to filter down jrnl entries to only four items with the following titles. There's no way for me to easily narrow down on say item two and four. I can use `-n 3` to see both but then I also get item three in the middle. 

A fuzzy search feature (`grep -i -E "Topic1|Topic3"`) where you could search on an entry's contents in addition to tags (either title only or anywhere in the entry) would be amazing. An interactive menu-system with `--menu` would also be amazing.

```sh
> jrnl @notes @keyword --short
2019-09-26 18:29 Lecture on Topic1
2019-09-27 19:27 Reading notes Topic2
2019-10-02 18:23 Topic3
2019-10-11 18:42 Project Design Topic4
```

**Comment by alichtman** [comment]
Sounds like a great use-case for `fzf`: https://github.com/nk412/pyfzf

**Comment by peter-schmidbauer** [comment]
pyfzf hasn't been updated in 2 years. I'd avoid using it.
There's a lot of interactive prompt packages out there, but I'd be careful jumping on one too quickly. A lot of them are unmaintained.

**Comment by alichtman** [comment]
>  There's a lot of interactive prompt packages out there, but I'd be careful jumping on one too quickly. 

I totally agree, but I haven't found one that seems better for this use-case.

**Comment by alichtman** [comment]
This should not be closed.

**Comment by wren** [comment]
@alichtman Agreed. Going through now after the holidays to clean up the backlog a bit. Thanks for pointing this one out!

**Comment by wren** [comment]
We now have `-contains` to search the body of journal entries, which can be used in combination with the other filters to refine your search.

Still not sure about an interactive menu option, but I'll leave this open while we consider it.
